mgmtsystem/hawkular.py

```diff
@@ -137,47 +137,55 @@ def list_server_deployment(self, feed_id, type_id='Deployment'):
         entities = []
         entities_j = self.api.get_json('feeds/{}/resourceTypes/{}/resources'
                                        .format(feed_id, type_id))
-        for entity_j in entities_j:
-            entity = Deployment(entity_j['id'], entity_j['name'], entity_j['path'])
-            entities.append(entity)
+        if entities_j is not None:
```

Comment:
This can just be ```if entities_j```, unless you are explicitly testing to see if it is not a ```None``` object. However given the loop that happens after, ```if entities_j``` is cleaner and more pythonic.